### PR TITLE
Migrate to Vue 3

### PR DESCRIPTION
**Pull request by haniAljti**
- upgraded vue 2 to vue 3 using vue 3 compat
- updated and added a wrapper for Ui-toast-editor since the wrapper does not support vue 3 but it should function the same way.
- there is no update for "vue2-interact" that works on vue 3 but i've built the same functionality using Hammerjs. (I tested it. but it would be better if anyone tested it more)
- the project uses pinia instead of vuex now. Which should also solve #212
- updated other dependancies to use vue 3

### CODE REVIEW

File: frontend/src/components/KickUserWrapper.vue [hunk 1]
```diff
@@ -58,15 +73,18 @@ export default Vue.extend({
   },
   methods: {
     openModal() {
-      this.$root.$emit("bv::hide::popover");
-      this.$root.$emit("bv::show::modal", `modal${this.member.memberID}`, "#btnShow");
+      this.showModal = true;
+      this.showPopover = true;
+      // this.$root.$emit("bv::hide::popover");
```
Comment (stritti) [inline frontend/src/components/KickUserWrapper.vue:78]: Not needed any more?

File: frontend/src/components/KickUserWrapper.vue [hunk 2]
```diff
@@ -58,15 +73,18 @@ export default Vue.extend({
...
+      // this.$root.$emit("bv::show::modal", `modal${this.member.memberID}`, "#btnShow");
     },
     closeModal() {
-      this.$root.$emit("bv::hide::modal", `modal${this.member.memberID}`, "#btnShow");
+      this.showModal = false;
+      // this.$root.$emit("bv::hide::modal", `modal${this.member.memberID}`, "#btnShow");
```
Comment (stritti) [inline frontend/src/components/KickUserWrapper.vue:83]: Not needed any more?

**Review by stritti — APPROVED** [review]
Just reviewed sources. Not tested live yet.

Really great work @haniAljti !

File: frontend/src/components/KickUserWrapper.vue (hunk 1 above)
Comment (haniAljti) [reply frontend/src/components/KickUserWrapper.vue]: you're right, i removed it.

File: frontend/src/components/KickUserWrapper.vue (hunk 2 above)
Comment (haniAljti) [reply frontend/src/components/KickUserWrapper.vue]: Removed

**Comment by SponsoredByPuma** [comment]
Hey @haniAljti,
nice work overall. 😄
The problems I've noticed are:
- the jira Component doesn't work anymore ( the b-modal is not showing up anymore)
- trying to create a session with imported user stories breaks the project, resulting in these error messages: 
image
- creating a session, where the host can vote as well does not work anymore 
image
(this is probably a problem of the props & params of the prepare & session page). All of the params are undefined in the session page

The rest looks good. Really nicely done :)

**Comment by haniAljti** [comment]
Thanks. I will take a look.

**Comment by haniAljti** [comment]
Hi @SponsoredByPuma, Please try if this fixes the issues you mentioned. Im not connected to Jira so i can't test that.

**Comment by SponsoredByPuma** [comment]
I had to remove the radio buttons on the prepare Page since i got the same bug as here: https://github.com/bootstrap-vue/bootstrap-vue/issues/7182

**Comment by SponsoredByPuma** [comment]
also had to remove the auto input component since selecting a project didnt work and also typing in the input field was impossible.

**Review by SponsoredByPuma — APPROVED** [review]

**Pull request merged**